src/tools/contextTool.ts

```diff
@@ -55,6 +66,88 @@ function normalizeContent(content: string): string {
   return content.replace(/\r\n/g, '\n').trim();
 }
 
+/**
+ * Parses the Handoff Mode from WorkflowContext.md content.
+ * Looks for "Handoff Mode: <mode>" line and extracts the mode value.
+ * 
+ * @param workflowContent - Raw WorkflowContext.md content
+ * @returns Parsed handoff mode or 'manual' as default
+ */
+export function parseHandoffMode(workflowContent: string): HandoffMode {
+  if (!workflowContent) {
+    return 'manual';
+  }
+
+  const match = workflowContent.match(HANDOFF_MODE_PATTERN);
+  if (match) {
+    return match[1].toLowerCase() as HandoffMode;
+  }
+
+  return 'manual';
+}
+
+/**
+ * Returns mode-specific handoff behavior instructions.
+ * These instructions tell the agent exactly how to behave when completing work,
+ * based on the handoff mode configured in WorkflowContext.md.
+ * 
+ * @param mode - The handoff mode (manual, semi-auto, or auto)
+ * @returns Markdown instructions for the agent
+ */
+export function getHandoffInstructions(mode: HandoffMode): string {
```

Comment:
Similar to how workItemInitPrompt.template.md has substantial prompt context in a markdown file for easier editing, I would like the manual, semi-auto, and auto handoff instructions to be stored in a template.md file.